.gitignore

```diff
@@ -30,3 +30,6 @@ address.json
 .DS_Store
 book
 .venv*
+
+# Temporary documentation
+/docs
```

Comment:
Unnecessary / ? Remove for consistency?

Reply:
Probably added it because it annoyed me while testing 😅 .
If you don't need it, I'm fine with that, but those docs if run locally will be added to the docs folder

Reply:
Ahh. Yeah of course. No problem
```suggestion
docs
```